DashKitUI/PumpManager/DashHUDProvider.swift

```diff
@@ -43,7 +43,7 @@ internal class DashHUDProvider: NSObject, HUDProvider {
         self.pumpManager.addPodStatusObserver(self, queue: .main)
     }
 
-    public func createHUDViews() -> [BaseHUDView] {
+    public func createHUDViews() -> [LevelHUDView] {
```

Comment:
Loop is never going to do anything with more than one view, right?  So this method signature should reflect that.